packages/mc-scripts/src/commands/login.js

```diff
@@ -0,0 +1,52 @@
+const { promisify } = require('util');
+const read = promisify(require('read'));
+const { processConfig } = require('@commercetools-frontend/application-config');
+const CredentialsStorage = require('../utils/credentials-storage');
+const authenticator = require('../utils/auth');
+const validateEmail = require('../utils/validate-email');
+
+const getEmail = async () => {
+  let email = await read({ prompt: 'Email: ' });
+  if (!validateEmail(email)) {
+    console.log('Please input a valid email');
+    email = await getEmail();
+  }
+  return email;
+};
+
+const getPassword = async () => {
+  return await read({ prompt: 'Password: ', silent: true });
+};
+
+const successfulLoginMessage = (cloudIdentifier) => {
+  console.log(
+    `You've successfully logged into the ${cloudIdentifier} environment.`
+  );
+  process.exit(0);
+};
+
+const login = async () => {
+  const applicationConfig = processConfig();
+  const cloudIdentifier = applicationConfig.env.location;
+  const credentialsStorage = new CredentialsStorage(cloudIdentifier);
```

Comment:
Nit: in general I would prefer to instantiate such objects on the module level rather than on each function execution.

The storage key can be passed to each instance method (like what you do with e.g. `localStorage`).

While in this specific case it's not a big deal, as this module is executed once anyway, I would still recommend to refactor it to get more in the habit of following these kind of best practices.